Project/target/classes/templates/login.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
```

Comment:
I thnk it might be because he first merged master into his own branch which is okay. Ensure this the same thing on master. If not, this should be another PR